maskrcnn_benchmark/modeling/roi_heads/mask_head/inference.py

```diff
@@ -38,17 +43,19 @@ def forward(self, x, boxes):
         mask_prob = x.sigmoid()
 
         # select masks coresponding to the predicted classes
-        num_masks = x.shape[0]
         labels = [bbox.get_field("labels") for bbox in boxes]
         labels = torch.cat(labels)
-        index = torch.arange(num_masks, device=labels.device)
+        index = arange_like(x)
```

Comment:
I would hope this doesn't have a performance impact.

Reply:
This is fine!